src/com/mendix/widget/carousel/components/__tests__/CarouselControl.spec.ts

```diff
@@ -0,0 +1,63 @@
+import { ShallowWrapper, shallow } from "enzyme";
+import { DOM, createElement } from "react";
+
+import { CarouselControl, CarouselControlProps } from "../CarouselControl";
+
+describe("CarouselControl", () => {
+
+    let carouselControl: ShallowWrapper<CarouselControlProps, any>;
+    let clickCount = 0;
+    let onClick = () => clickCount++;
```

Comment:
If you replace this with a spy (created in the `beforeEach`), you can simply verify that it has been called later on. See my comments later :)